data/ignition/systemd/units/create-cluster-and-infra-env.service

```diff
@@ -9,6 +10,7 @@ ExecStart=/usr/local/bin/create-cluster-and-infra-env.sh
 
 KillMode=none
 Type=oneshot
+RemainAfterExit=true
```

Comment:
Being that is listed as PartOf, will that not trigger the creation if the assisted-service-pod restarts? Will this be prevented by RemainAfterExit? If not, we'll probably need to have the logic for checking if it is already done in `create-cluster-and-infra-env.sh` (which is not a bad idea)

Reply:
Being `PartOf` has no real effect unless `RemainAfterExit` is also true (i.e. no restart is triggered by dependencies if a unit has already completed). Setting both ensures that this script gets re-run if the assisted-service pod restarts, which is necessary because a pod restart also clears the database.